### PR TITLE
docs: developer guide datanode overview

### PR DESCRIPTION
**Pull request by fengjiachun**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/en/v0.4/developer-guide/datanode/overview.md [hunk 1]
```diff
@@ -2,47 +2,28 @@
 
 ## Introduction
 
-`Datanode` is mainly responsible for storing the actual data for GreptimeDB. `Datanode` itself is a
-stand-alone database service. Meanwhile, it could cooperate with `Frontend` and `Meta` to form a
-distributed database service, as the following picture shows:
+`Datanode` is mainly responsible for storing the actual data for GreptimeDB.. As we know, in GreptimeDB,
```
Comment (xtang) [inline docs/en/v0.4/developer-guide/datanode/overview.md:5]:
```suggestion
`Datanode` is mainly responsible for storing the actual data for GreptimeDB. As we know, in GreptimeDB,
```

**Review by xtang — APPROVED** [review]

**Review by beryl678 — APPROVED** [review]

File: docs/public/datanode.png
[the remark this answers is not in the document]
Comment (xtang) [reply docs/public/datanode.png:1]: @fengjiachun PTAL

[the remark this answers is not in the document]
Comment (nicecui) [reply docs/public/datanode.png:1]: Please use #6F6FFF and #8322FF for line colors.

[the remark this answers is not in the document]
Comment (fengjiachun) [reply docs/public/datanode.png:1]: I will fix it ASAP

**Review by nicecui — APPROVED** [review]

**Review by zhaoyingnan01 — APPROVED** [review]

**Pull request merged**